setup.py

```diff
@@ -10,7 +10,7 @@ def readme():
 
 setup(
     name='cdp-client',
-    version='1.2.1',
+    version='1.3.1',
```

Comment:
You changed the API so much that I think we should increase the major version here. The changes are not backwards compatible with old client code, right?

Reply:
Changed it to 2.0.0 then